plugins/statistic.koplugin/main.lua

```diff
@@ -0,0 +1,423 @@
+            return _("Enabled")
+        end,
+        checked_func = function() return self.is_enabled end,
+        callback = function()
+            -- if was enabled, have to save data to file
+            if self.last_time and self.is_enabled then
+                self:exportToFile(self:getBookProperties())
+            end
+
+            self.is_enabled = not self.is_enabled
+            -- if was disabled have to get data from file
+            if self.is_enabled then
+                self:initData()
+            end
+            self:onSaveSettings()
+        end,
+    }
+end
+
+function ReaderStatistic:getStatisticSettingsMenuTable()
+    return {
+        text_func = function()
+            return _("Settings")
+        end,
+        checked_func = function() return false end,
+        callback = function()
+            self:updateSettings()
+        end,
+    }
+end
+
+function ReaderStatistic:updateSettings()
+    self.settings_dialog = MultiInputDialog:new {
+        title = _("Statistic settings"),
+        fields = {
+            {
+                text = "",
+                input_type = "number",
+                hint = _("Min seconds, default is 5"),
+            },
+            {
+                text = "",
+                input_type = "number",
+                hint = _("Max seconds, default is 90"),
+            },
+        },
+        buttons = {
+            {
+                {
+                    text = _("Cancel"),
+                    callback = function()
+                        self.settings_dialog:onClose()
+                        UIManager:close(self.settings_dialog)
+                    end
+                },
+                {
+                    text = _("Update"),
+                    callback = function()
+                        self.settings_dialog:onClose()
+                        UIManager:close(self.settings_dialog)
+                        self:onSaveSettings(MultiInputDialog:getFields())
+                    end
+                },
+            },
+        },
+        width = Screen:getWidth() * 0.95,
+        height = Screen:getHeight() * 0.2,
+        input_type = "number",
+    }
+    self.settings_dialog:onShowKeyboard()
+    UIManager:show(self.settings_dialog)
+end
+
+function ReaderStatistic:getStatisticForCurrentBookMenuTable()
+    self.status_menu = {}
+
+    local book_status = Menu:new {
+        title = _("Status"),
+        item_table = self:updateCurrentStat(),
+        is_borderless = true,
+        is_popout = false,
+        is_enable_shortcut = false,
+        width = Screen:getWidth(),
+        height = Screen:getHeight(),
+        cface = Font:getFace("cfont", 20),
+    }
+
+    self.status_menu = CenterContainer:new {
+        dimen = Screen:getSize(),
+        book_status,
+    }
+
+    book_status.close_callback = function()
+        UIManager:close(self.status_menu)
+    end
+
+    book_status.show_parent = self.status_menu
+
+    return {
+        text = "Current",
```

Comment:
No, encapsulate in _()

Cheers

On September 9, 2015 8:32:25 PM GMT+02:00, Alexander Pletnev <notifications@github.com> wrote:
>> +        cface = Font:getFace("cfont", 20),
>> +    }
>> +
>> +    self.status_menu = CenterContainer:new {
>> +        dimen = Screen:getSize(),
>> +        book_status,
>> +    }
>> +
>> +    book_status.close_callback = function()
>> +        UIManager:close(self.status_menu)
>> +    end
>> +
>> +    book_status.show_parent = self.status_menu
>> +
>> +    return {
>> +        text = "Current",
>
>For add translation i have to execute "make pot"? 
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/koreader/koreader/pull/1628/files#r39079192

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.